custom/icds/repeaters/phi.py

```diff
@@ -0,0 +1,47 @@
+from django.utils.translation import ugettext_lazy as _
+
+from corehq import toggles
+from corehq.motech.repeaters.models import CaseRepeater
+from custom.icds.integrations.phi import send_request
+from custom.icds.repeaters.generators.phi import (
+    SearchByParamsPayloadGenerator,
+    ValidatePHIDPayloadGenerator,
+)
+
+
+class BasePHIRepeater(CaseRepeater):
+    @classmethod
+    def available_for_domain(cls, domain):
+        return toggles.PHI_CAS_INTEGRATION.enabled(domain)
+
+
+class SearchByParamsRepeater(BasePHIRepeater):
+    payload_generator_classes = (SearchByParamsPayloadGenerator,)
+    friendly_name = _("Search for Beneficiary via params to get PHI ID")
+
+    def allowed_to_forward(self, payload):
+        return (
+            super(SearchByParamsRepeater, self).allowed_to_forward(payload) and
```

Comment:
This isn't a request to change, just a tip: Now that we live in the glorious Python 3 future, you can write this as
```python
    super().allowed_to_forward(payload)
```
:tada: Yay!